attribute-service-impl/src/test/java/org/hypertrace/core/attribute/service/AttributeServiceImplTest.java

```diff
@@ -200,14 +201,18 @@ public void testFindAttributes() {
 
       Filter filter = queryCaptor.getValue().getFilter();
       // The structure of the filters is an and(^) filter chain that looks like this:
-      // (((tenant_id ^ fqn) ^ (scope_string | scope)) ^ key)
+      // ((((tenant_id ^ fqn) ^ (scope_string | scope)) ^ key) ^ internal)
```

Comment:
The test is monster big, anyways I wonder if we test both scenarios where internal is present and when it isn't (as it is optional).

Reply:
I have added a separate test for the scenario when internal is not present.

Reply:
I am curious where exactly